lib/simulator-xcode-6.js

```diff
@@ -174,11 +174,14 @@ class SimulatorXcode6 {
       return path.resolve(this.getDir(), s);
     });
 
-    let existence = await asyncmap(files, async (f) => { return fs.hasAccess(f); });
-
-    existence = await B.all(existence); // will throw an error if an fs.stat call fails
+    let existences = await asyncmap(files, async (f) => { return fs.hasAccess(f); });
+    for (let i = 0; i < files.length; i++) {
```

Comment:
Run locally yes. I ran the tests locally myself and they failed, so there's
work to do. Could be related to my updating to El Capitan

On Wed, Jan 6, 2016 at 10:36 AM, Jonah Stiennon <jonahss@gmail.com> wrote:

> Yeah just style, no real advantages.
>
> On Wed, Jan 6, 2016 at 10:15 AM, Jonathan Lipps <notifications@github.com>
> wrote:
>
>> In lib/simulator-xcode-6.js
>> <https://github.com/appium/appium-ios-simulator/pull/39#discussion_r48989050>
>> :
>>
>> > @@ -174,11 +174,14 @@ class SimulatorXcode6 {
>> >        return path.resolve(this.getDir(), s);
>> >      });
>> >
>> > -    let existence = await asyncmap(files, async (f) => { return fs.hasAccess(f); });
>> > -
>> > -    existence = await B.all(existence); // will throw an error if an fs.stat call fails
>> > +    let existences = await asyncmap(files, async (f) => { return fs.hasAccess(f); });
>> > +    for (let i = 0; i < files.length; i++) {
>>
>> any advantages to using Array.forEach? I could also use for...of which
>> is more our style I suppose.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/appium/appium-ios-simulator/pull/39/files#r48989050>.
>>
>
>